spring-rabbit/src/test/java/org/springframework/amqp/rabbit/AsyncRabbitTemplateTests.java

```diff
@@ -336,12 +420,12 @@ public ConnectionFactory connectionFactory() {
 
 		@Bean
 		public Queue requests() {
-			return new AnonymousQueue();
+			return new AnonymousQueue(new AnonymousQueue.Base64UrlNamingStrategy());
 		}
 
 		@Bean
 		public Queue replies() {
-			return new AnonymousQueue();
+			return new AnonymousQueue(new AnonymousQueue.Base64UrlNamingStrategy());
```

Comment:
I think we can revert this change already since `AnonymousQueue` is now Base64 by default.